e2e-tests/cypress/integration/appeal-submission-appeal-site-address/appeal-submission-appeal-site-address.js

```diff
@@ -0,0 +1,122 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('the user is prompted for the site address', () => {
+  cy.goToSiteAddressPage();
+});
+
+When(
+  'the user provides their appeal site address as {string} and {string} and {string} and {string} and {string}',
+  (addressLine1, addressLine2, townCity, county, postcode) => {
+    cy.provideAddressLine1(addressLine1);
+    cy.provideAddressLine2(addressLine2);
+    cy.provideTownOrCity(townCity);
+    cy.provideCounty(county);
+    cy.providePostcode(postcode);
+    cy.clickSaveAndContinue();
+  },
+);
+
+When('the user provides a value which is too long - {string} : {int}', (component, count) => {
+  const value = Array(count + 1).join('x');
```

Comment:
Is this the same as `'x'.repeat(count + 1)`

Reply:
Maybe lol